models/student-group.model.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const StudentGroupSchema = new Schema(
+  {
+    groupID: Number,
+    groupName: String,
+    topic: String,
```

Comment:
Remove the topic from here. I will implement another model. There need to be some other attributes for the topic aswell.